<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Option</class>
 <widget class="QWidget" name="Option">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>625</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="glay_main">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QFrame" name="fr_main">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="vlay_fr_main">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="fr_menu_top">
        <layout class="QHBoxLayout" name="hlay_menu_top">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lb_mt_ico"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lb_mt_nom"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_mt_quitter"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_body">
        <layout class="QHBoxLayout" name="hlay_opt_body">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="trw_option">
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <item>
            <property name="text">
             <string>Configs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Thèmes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Infos</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stk_option">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pg_opt_configs">
            <layout class="QVBoxLayout" name="vlay_pg_opt_configs">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QFontComboBox" name="fcb_opt_ft_font"/>
             </item>
             <item>
              <widget class="QFrame" name="fr_opt_cfg_opacity">
               <layout class="QHBoxLayout" name="hlay_opt_fr_opt_cfg_opacity">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="lb_opt_cfg_opacity">
                  <property name="text">
                   <string>Opacité </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="sb_opt_cfg_opacity"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="fr_opt_cfg_autoclose">
               <layout class="QGridLayout" name="glay_fr_opt_cfg_autoclose">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <property name="spacing">
                 <number>5</number>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="lb_opt_cfg_autoclose">
                  <property name="text">
                   <string>Auto close : </string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="lb_opt_cfg_debug">
                  <property name="text">
                   <string>Debug : </string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="ck_opt_cfg_autoclose"/>
                </item>
                <item row="0" column="1">
                 <widget class="QCheckBox" name="ck_opt_cfg_debug"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="lb_opt_cfg_ui_pin">
                  <property name="text">
                   <string>ToolBox Pin :</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QCheckBox" name="ck_opt_cfg_ui_pin"/>
                </item>
                <item row="0" column="2" rowspan="3">
                 <spacer name="horizontalSpacer_10">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="fr_opt_cfg_resize">
               <layout class="QGridLayout" name="glay_fr_opt_cfg_resize">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <property name="spacing">
                 <number>5</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="lb_opt_cfg_resize">
                  <property name="text">
                   <string>Manuel : </string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QCheckBox" name="ck_opt_cfg_resize"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="lb_opt_cfg_resize_width">
                  <property name="text">
                   <string>Largeur : </string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="lb_opt_cfg_resize_height">
                  <property name="text">
                   <string>Hauteur : </string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="sb_opt_cfg_resize_width"/>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="sb_opt_cfg_resize_height"/>
                </item>
                <item row="0" column="2" rowspan="3">
                 <spacer name="horizontalSpacer_11">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_14">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pg_opt_themes">
            <layout class="QVBoxLayout" name="vlay_pg_opt_themes">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QComboBox" name="cb_opt_tm_theme"/>
             </item>
             <item>
              <spacer name="verticalSpacer_15">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="hlay_opt_tm_colors">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pb_opt_tm_th1">
                 <property name="text">
                  <string>TH1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pb_opt_tm_th2">
                 <property name="text">
                  <string>TH2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pb_opt_tm_th3">
                 <property name="text">
                  <string>TH3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pb_opt_tm_bn1">
                 <property name="text">
                  <string>BN1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pb_opt_tm_bn2">
                 <property name="text">
                  <string>BN2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_16">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pg_opt_infos">
            <layout class="QVBoxLayout" name="vlay_pg_opt_infos">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="lb_opt_info_nom"/>
             </item>
             <item>
              <spacer name="verticalSpacer_17">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>80</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="lb_opt_info_desc"/>
             </item>
             <item>
              <spacer name="verticalSpacer_18">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="lb_opt_info_auteur"/>
             </item>
             <item>
              <widget class="QLabel" name="lb_opt_info_version"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_opt_bottom">
        <layout class="QHBoxLayout" name="hlay_opt_bottom">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_opt_appliquer"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_opt_ok"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
